### PR TITLE
Update units-turtle.lua

### PR DESCRIPTION
**Pull request by Plan414**
1. Added missing coordinated for NPC Tansy Sparkpen in Stormwind.
2. Fixed coordinated for NPC High Elf Ranger in Stormwind.
3. Fixed coordinates for NPC Ranger Canarah Kim'Alah in Stormwind.
4. Fix coordinates for NPC Applebough in Stormwind.

### CODE REVIEW

File: db/units-turtle.lua [hunk 1]
```diff
@@ -4470,6 +4470,9 @@ pfDB["units"]["data-turtle"] = {
   [50760] = {
     ["fac"] = "A",
     ["lvl"] = "30",
+    ["coords"] = {
+      [1] = { 64.9, 73.1, 1519, 0 }
+	},
```
Comment (shagu) [inline db/units-turtle.lua:4475]: Can you please fix the identation of this line? :)

**Comment by shagu** [comment]
Thanks for your PR :)

@Gurky-Kronos Can you please review these stormwind positions? I don't know where the initial values came from.

**Comment by Gurky-Kronos** [comment]
I think some ended up wrong because later changes to the map so I am assuming these are more correct. I noticed the other day when leveling a lot are moved up a bit from where they should be. Also there are some minimap issues in stormwind so this could be the cause as well.

**Comment by Gurky-Kronos** [comment]
@shagu I would say this can't hurt much to merge

**Comment by shagu** [comment]
Thanks :+1:

**Pull request merged**